packages/steiger-plugin-fsd/src/no-layer-public-api/README.md

```diff
@@ -11,6 +11,8 @@ According to the _public API rule on slices_:
 
 A corollary to this rule is that the layer itself should not have an index file.
 
+**Exception:** index files are allowed on `app` layer because it's the entry point of the application.
```

Comment:
I would rephrase this to say "because some people prefer to have that as their app's entrypoint" to emphasize that this is not a rule, but a preference